types/firestore.d.ts

```diff
@@ -143,6 +143,13 @@ declare namespace FirebaseFirestore {
      * when the client becomes idle. Defaults to 1.
      */
     maxIdleChannels?: number;
+
+    /**
+     * Whether to skip nested properties that are set to `undefined` during object
+     * serialization. If set to `false` or omitted, the SDK will throw an
+     * exception when it encounters values of type `undefined`.
```

Comment:
Do you think we should also document the effects (or non-effect) during deserialization?

Reply:
Sounds good. Changed to:

```
  /**
   * Whether to skip nested properties that are set to `undefined` during object
   * serialization. If set to `true`, these properties will be skipped and are 
   * not be written to Firestore. If the setting is set `false` or omitted, 
   * the SDK will throw an exception when it encounters properties of type 
   * `undefined`.
   */
```